pkgs/tools/networking/openssh/default.nix

```diff
@@ -66,5 +82,6 @@ in
     ];
 
     extraNativeBuildInputs = [ autoreconfHook ];
+    extraMeta.knownVulnerabilities = [ "CVE-2023-25136" ];
```

Comment:
I think this is false. The advisory claims the vulnerability was introduced in 9.1, this one is still on 9.0 so it seems to have dodged it.

Reply:
Alright, I will kill the package with the next real vulnerability